README.md

```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
 You can reach its socket at `/tmp/unit-control/control.unit.sock` assuming no
 further customizations have been made.
 
-### Amazon Linux, Fedora, Red Hat
+### Debian, Ubuntu, Amazon Linux, Fedora, Red Hat
 
+This helper script configures the correct package repositories for system.
 ``` console
 $ wget https://raw.githubusercontent.com/nginx/unit/master/tools/setup-unit && chmod +x setup-unit
-# ./setup-unit repo-config && yum install unit
-# ./setup-unit welcome
+# ./setup-unit repo-config
+```
+
+Debian derivatives:
+``` console
+# apt install unit
+```
+
+RHEL derivatives:
+``` console
+# yum install unit
 ```
 
 For details and available language packages, see the
 [docs](https://unit.nginx.org/installation/#official-packages).
 
+## Getting Started with `unitctl`
+
+[`unitctl`](tools/README.md) streamlines the management of NGINX Unit processes
+through an easy-to-use command line interface. To get started with `unitctl`,
+download it from the
+[official GitHub releases](https://github.com/nginx/unit/releases)
+or [Homebrew](#macos).
+
+### Installation
+
+> [!NOTE]
+> If you installed Unit with [Homebrew](#macos), you can skip this step
+> as `unitctl` is included by default.
 
-### Debian, Ubuntu
+Download the appropriate `unitctl` binary for your system from the
+[NGINX Unit releases](https://github.com/nginx/unit/releases/).
 
 ``` console
-$ wget https://raw.githubusercontent.com/nginx/unit/master/tools/setup-unit && chmod +x setup-unit
-# ./setup-unit repo-config && apt install unit
-# ./setup-unit welcome
+$ tar xzvf unitctl-master-x86_64-unknown-linux-gnu.tar.gz
+# mv unitctl /usr/local/bin/
 ```
 
-For details and available language packages, see the
-[docs](https://unit.nginx.org/installation/#official-packages).
 
-## Configuration
+## Launch Unit using Docker
+If you have [Docker installed](https://docs.docker.com/engine/install/) on
+your machine, and then you can effortlessly spin up one of
+[Unit's official Docker images](https://hub.docker.com/_/unit)
+alongside your application.
+
+> [!TIP]
+> How-to and configuration guides are available on
+[unit.nginx.org](https://unit.nginx.org/howto/) for web application frameworks
+built with Python, PHP, WebAssembly, Node.js, Ruby, and more.
 
-NGINX Unit provides a RESTful API for dynamic configuration.
-See the [control API documentation](https://unit.nginx.org/controlapi/)
-for more information on what endpoints are available and how to use them.
+Here's an example using the `unit:python` Docker image:
+```
+unitctl instances new 127.0.0.1:8001 /path/to/app 'unit:python'
+```
```

Comment:
To be consistent this snippet should be of type `console` and could you put the prompt character in to show who it's intended that the command be run as.